app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteView.scss

```diff
@@ -63,12 +63,12 @@
 	background-image: buildAdBlockActiveIcon(#1dafed);
 }
 
-// Smart Blocking Icon
+// Smart Browsing Icon
 .SetupAntiSuite__feature .smart-blocking {
```

Comment:
Let's also change the class names `.smart-blocking` -> `.smart-browsing`

Reply:
Updated the class names!